### PR TITLE
Ensure a working results sort, and rename functions to match behavior

### PR DESCRIPTION
**Pull request by hamstap85**

### CODE REVIEW

File: brooklinevoiceapp/mycity/utils/address_utils.py [hunk 1]
```diff
@@ -70,34 +71,8 @@ def get_closest_result(home_coordinates, features):
...
-            feature_coordinates
-        )
-        if distance < closest['distance']:
-            closest = {
-                'feature': feature,
-                'distance': distance
-            }
-
-    return closest['feature']
+    return sorted(features, key=lambda feature: get_distance(home_coordinates, feature['geometry']))
```
Comment (Joshhw) [inline brooklinevoiceapp/mycity/utils/address_utils.py]: is this a function we created?

**Comment by jmartini** [comment]
I have a fix to the unit tests on this branch, but can't push since the PR since its on Hameed's github. The fix is here: 9c65caaf1bb1ac02152d601237efda68c8f8b5e4

**Comment by hamstap85** [comment]
@jmartini Done. Also didn't finish my debugging last night because I crashed

File: brooklinevoiceapp/mycity/utils/brookline_arcgis_api_utils.py [hunk 2]
```diff
@@ -111,58 +116,55 @@ def get_nearest_feature_json(address: str,
         INSR_PARAM: "102100",
         OUT_FIELDS_PARAM: "*",
         OUTSR_PARAM: "102100",
-        GEOMETRY_PARAM: json.dumps(_geocode_address(address))
+        GEOMETRY_PARAM: json.dumps(home_address)
```
Comment (Joshhw) [inline brooklinevoiceapp/mycity/utils/brookline_arcgis_api_utils.py]: This needs to be changed to:
`GEOMETRY_PARAM: json.dumps([home_address['x'], home_address['y']])`

the reason being is coordinates being passed aren't in the format they need to be. This fixes that issue.

Comment (Joshhw) [reply brooklinevoiceapp/mycity/utils/brookline_arcgis_api_utils.py]: This should fix it according to my testing.

**Comment by hamstap85** [comment]
@Joshhw @jmartini Done FINALLY OH MY GOD

**Comment by jmartini** [comment]
This does not work for the trash day intent:

Repro: 
1. Ask "Open Town of Brookline"
2. Ask " trash day at 333 washington street"

Results:
"There was a problem with the requested skill's response"

> 'x': KeyError
Traceback (most recent call last):
  File "/var/task/lambda_function.py", line 34, in lambda_handler
    response = mycity_response_to_platform(execute_request(model))
  File "/var/task/brookline_controller.py", line 27, in execute_request
    response = on_intent(mycity_request)
  File "/var/task/brookline_controller.py", line 109, in on_intent
    return get_trash_pickup_info(mycity_request)
  File "/var/task/mycity/intents/trash_day_intent.py", line 38, in get_trash_pickup_info
    response.output_speech = _get_output_speech_for_address(current_address)
  File "/var/task/mycity/intents/trash_day_intent.py", line 51, in _get_output_speech_for_address
    response = get_trash_day_json(address)
  File "/var/task/mycity/utils/brookline_arcgis_api_utils.py", line 170, in get_trash_day_json
    return _get_sorted_features_json(address, MapFeatureID.TRASH_DAY)
  File "/var/task/mycity/utils/brookline_arcgis_api_utils.py", line 126, in get_sorted_features_json
    return address_utils.get_sorted_features(home_address, features)
  File "/var/task/mycity/utils/address_utils.py", line 84, in get_sorted_features
    return sorted(features, key=lambda feature: get_distance(home_coordinates, feature['geometry']))
  File "/var/task/mycity/utils/address_utils.py", line 84, in <lambda>
    return sorted(features, key=lambda feature: get_distance(home_coordinates, feature['geometry']))
  File "/var/task/mycity/utils/address_utils.py", line 59, in get_distance
    lon2 = end['x']
KeyError: 'x'

**Comment by jmartini** [comment]
@hamstap85 can this be closed now? I think these changes were done in #41

**Comment by hamstap85** [comment]
@jmartini Yes of course. Thanks for catching this.

**Pull request closed without merging**